specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/SavedSearches.json

```diff
@@ -234,6 +234,14 @@
           "type": "string",
           "description": "The query expression for the saved search. Please see https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-search-reference for reference."
         },
+        "functionAlias": {
+          "type": "string",
+          "description": "The function alias if query serves as a function."
+        },
+        "functionParameters": {
+          "type": "string",
+          "description": "The optional function parameters if query serves as a function. For proper syntax please refer to https://docs.microsoft.com/en-us/azure/kusto/query/functions/user-defined-functions."
```

Comment:
I would add the syntax here, and send them out for more info. Maybe something like this:
Expected value is in the following format: "param-name1:type1 = default-value1, param-name2:type2 = default-value2, ..."